### PR TITLE
feat: patch account and access keys added

### PR DESCRIPTION
**Pull request by Buckram123**
Trying to implement #19. Not sure, how to make a friendly API for this. Thinking of using builder here.

### CODE REVIEW

File: workspaces/src/worker/impls.rs [hunk 1]
```diff
@@ -186,6 +187,21 @@ impl Worker<Sandbox> {
...
 
+    /// Patch account into the sandbox network, given an optional Account fields. This will allow
+    /// us to set amount or locked amount of NEAR tokens, change code hash, and storage usage.
+    pub async fn patch_account(
+        &self,
+        account_id: &AccountId,
+        amount: Option<Balance>,
+        locked: Option<Balance>,
+        code_hash: Option<CryptoHash>,
+        storage_usage: Option<StorageUsage>,
```
Comment (ChaoticTempest) [inline workspaces/src/worker/impls.rs:198]: > Not sure, how to make a friendly API for this. Thinking of using builder here.

yeah definitely, would be great if we can get a builder going for this. You can take a look at [patch.rs](https://github.com/near/workspaces-rs/blob/main/workspaces/src/rpc/patch.rs) to see how to get one going. Probably also best to put it there as well

File: workspaces/src/network/sandbox.rs [hunk 2]
```diff
@@ -158,6 +162,70 @@ impl Sandbox {
         Ok(())
     }
 
+    pub(crate) async fn patch_account(
```
Comment (ChaoticTempest) [inline workspaces/src/network/sandbox.rs:165]: Can you add tests for this and `patch_access_key`. Let me know if they aren't as straightforward to implement, but they should be simple if you follow the ones in [tests/patch_state.rs](https://github.com/near/workspaces-rs/blob/main/workspaces/tests/patch_state.rs).

**Review by ChaoticTempest — COMMENTED** [review]
Thanks so much for implementing this! Just some minor things to add

**Comment by Buckram123** [comment]
Closing in favor of #124.

**Pull request closed without merging**